recipes/systemc-cci/1.0.0/CMakeLists.txt

```
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(cci)
```

```suggestion
project(systemc CXX)
```
enforce the language